### DISCUSSION TITLE
Playground on codepen not working

### DISCUSSION THREAD

**Discussion by nelsonmic** [ThinkUniform/uniformcss#5] [Q&A] [answered]
Hello there! I was playing around with uniform CSS, just to get a feel of it and see how it works, but I dont think the playground featured on the website is working, i was wondering if that's intentional?

**Comment by jinsupark** [comment, marked as answer]
Hey! Thanks for checking this out, try this link https://codepen.io/UniformCSS/pen/poNNqaE

It should be working now!

**Reply by nelsonmic** [reply]
Yeah, it's working now, awesome!

**Reply by jinsupark** [reply]
If you have any questions about this project please don't hesitate!
